ts/screens/onboarding/OnboardingCompletedScreen.tsx

```diff
@@ -28,38 +22,30 @@ const OnboardingCompletedScreen = () => {
   const idp = O.isSome(idpSelected) ? idpSelected.value.name : "";
   const route = useRoute();
 
-  const continueButtonProps: BlockButtonProps = {
-    bordered: false,
-    title: I18n.t("global.buttons.continue"),
-    onPress: () => {
-      trackLoginEnded(
-        isFastLoginEnabled,
-        idp,
-        getFlowType(false, true),
-        route.name
-      );
-      dispatch(completeOnboarding());
-    }
-  };
-
   useOnFirstRender(() => {
     trackThankYouPageScreen();
   });
 
-  return (
-    <BaseScreenComponent>
-      <SafeAreaView style={IOStyles.flex}>
-        <InfoScreenComponent
-          image={<Pictogram name="fireworks" />}
-          title={I18n.t("onboarding.thankYouPage.title")}
-        />
+  const handleContinue = () => {
```

Comment:
What about using a callback?

Reply:
Interesting suggestion.
Can you explain why you think this will be usefull?

Reply:
Good catch! Actually here is not very useful and add a little overhead. Better to use `useCallback` when we use the function inside an `effect` or passing it on a `memoized` component.

Reply:
Good catch! Actually here is not very useful and add a little overhead. Better to use `useCallback` when we use the function inside an `effect` or passing it on a `memoized` component.

Reply:
I confirm that I didn't use `usecallback` in this case because is was not too relevant. This function is very simple and has no dependencies! 😁